packages/core/package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "build": "npm run build:cjs && npm run build:esm",
     "build:cjs": "tsc -b",
-    "build:esm": "ttsc -b tsconfig.esm.json"
+    "build:esm": "ttsc -b tsconfig.esm.json",
+    "build:client": "graphql-codegen -c graphql/codegen.yml"
```

Comment:
currently pretty "manual" but we could defs add this as a ci step in the future.